tests/unit/test_charm.py

```diff
@@ -109,6 +109,9 @@ def test_ready(self):
                         "OAUTH_DOMAIN": None,
                         "OAUTH_ADMIN_EMAIL": "admin@superset.com",
                         "SELF_REGISTRATION_ROLE": "Public",
+                        "HTTP_PROXY": None,
```

Comment:
Did you want to add any values here to make sure they come through as part of the test?